config/locales/en.yml

```diff
@@ -33,9 +33,10 @@ en:
         description: Description
         description_en: English description
         description_nl: Dutch description
+        disclaimer: Data on this activity may be outdated, as it was addes as a one-time copy of the information given at that time. Up-to-date info can be found on Koala.
```

Comment:
Is this translation used anywhere still?

Reply:
It is! In the description of an activity